.github/workflows/licensecheck.yml

```diff
@@ -0,0 +1,42 @@
+# This workflow will build a Java project with Maven
+# For more information see: https://help.github.com/actions/language-and-framework-guides/building-and-testing-java-with-maven
+
+name: License check
+
+on:
+  push:
+    branches: 
+      - 'master'
+      - 'tycho-*'
+  pull_request:
+    branches: 
+     - 'master'
+     - 'tycho-*'
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up JDK 8
```

Comment:
I think jDK8 is not required for the verification isn't it?

Reply:
Copy/paste and I thought I deleted useless stuff. Thanks for detailed review.